dbcon/joblist/limitedorderby.cpp

```diff
@@ -181,7 +208,7 @@ void LimitedOrderBy::finalize()
     {
         // *DRRTUY Very memory intensive. CS needs to account active
         // memory only and release memory if needed.
-        uint64_t memSizeInc = fRowsPerRG * fRowGroup.getRowSize();
+        uint64_t memSizeInc = fRowGroup.getSizeWithStrings();
```

Comment:
I'm curios why do you account the header size also? I believe we reuse an existing RowGroup just applying RGData to it so that we don't need to count the header size every time.

Reply:
Yes, I missed that point.